llvm/test/SYCLLowerIR/esimd_global.ll

```diff
@@ -6,18 +6,18 @@
 target datalayout = "e-i64:64-v16:16-v24:32-v32:32-v48:64-v96:128-v192:256-v256:256-v512:512-v1024:1024-n8:16:32:64"
 target triple = "spir64-unknown-unknown-sycldevice"
 
-%"class._ZTSN2cl4sycl5INTEL3gpu4simdIiLi16EEE.cl::sycl::INTEL::gpu::simd" = type { <16 x i32> }
+%"class._ZTSN2cl4sycl3ext5intel3gpu4simdIiLi16EEE.cl::sycl::ext::intel::gpu::simd" = type { <16 x i32> }
```

Comment:
no need to change here. This is incorrect name anyway (see correct one in my comment above). The type names are meaningless in LLVM IR, just serve as symbolic IDs.

Reply:
Mb we can introduce the replacement from INTEL to ext::intel in this patch, and after that these functions w incorrect names will be replaced with the correct ones as separate PR?

Reply:
Fixed